ex3/Premier league.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<p>Please plot the graphs using pandas (under the hood, pandas is using matplotlib in order to plot graphs).</p><p>Look for a solution of the following form - </p><p><code>df.groupby('Season')....apply....plot()</code></p><p> The utility <code>groupby</code>  is powerful for aggregated plots - learn to use it !</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>